backend/webapi/Data/Migrations/20241114233713_Delete PRP authorized licence table.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Pidp.Data.Migrations
+{
+    /// <inheritdoc />
+    public partial class DeletePRPauthorizedlicencetable : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+
```

Comment:
There seems to be an issue with your migration; there are no changes here in the Up/Down. Could you please regenerate the migration?

Also, please use CamelCasing when naming your migration.

Reply:
Fixed.